src/structure.cpp

```diff
@@ -227,6 +227,84 @@ void structure::choose_chunkdivision(const grid_volume &thegv, int desired_num_c
   }
 }
 
+std::vector<grid_volume> choose_chunkdivision(const grid_volume &thegv, int desired_num_chunks,
+                                              const symmetry &S) {
```

Comment:
This should really replace the old function — refactoring, not duplication.

Reply:
`meep::choose_chunkdivision` has been refactored to remove duplication from `structure::choose_chunkdivision` as much as possible.